api/config.js

```diff
@@ -22,5 +22,7 @@ module.exports = {
   scriptRunnerEndpoint:
     process.env.SCRIPT_RUNNER_ENDPOINT || 'http://localhost:9000',
   scriptRunnerAuthenticationToken:
-    process.env.AUTHENTICATION_TOKEN || secrets.read('AUTHENTICATION_TOKEN')
+    process.env.AUTHENTICATION_TOKEN || secrets.read('AUTHENTICATION_TOKEN'),
+  pepipostAPIKey:
```

Comment:
Pepipost? :rofl: Sounds so funny!

Reply:
Never heard of it O.o

Reply:
I looked for the cheapest provider. It's not really high tech to send emails.